native/e2e/recorders/ios-recorder.js

```diff
@@ -27,6 +27,10 @@ const recordSimulator = fileName =>
     fileName,
   ]);
 
+const optimiseRecordedVideo = filename => {
+  exec(`scripts/video/convert-to-mp4.sh ${filename}`);
```

Comment:
should this be execSync just to ensure that the video conversion is complete by the time the process exits, or are we confident that the script runs perfectly separately?

Reply:
Also should we change the name of this folder to `detox` instead of `video` so that it matches the commands we run

Or maybe we're planning to change the npm commands to `generate-videos` or something?

Reply:
> should this be execSync just to ensure that the video conversion is complete by the time the process exits, or are we confident that the script runs perfectly separately?

Yeah could do - maybe sync is better here. @matteo-hertel what do you think?

Reply:
> Also should we change the name of this folder to detox instead of video so that it matches the commands we run

Good shout.

Reply:
Either `exec` or `execSync` works as long as some sort of error handling is provided, for instance while testing my ffmpeg failed and no error was surfaced in the console

so something like
```
exec(script, (error, stdout, stderr) => {
  if (error) {
    console.error(`😱 : ${error}`);
    return;
  }`);
});
```

or if you want to go above an beyond use spawn

```
const { spawn } = require('child_process');
const awesomeScript = spawn('script', ['flags']);

awesomeScript.stderr.on('data', (data) => {
  console.log(`stderr: ${data}`);
});
```

Reply:
Actually, ignore that, I've added some utils to help with that 
https://github.com/Skyscanner/backpack/blob/master/native/e2e/utils.js#L29

you can use `command` for a promise base `spawn` or `rawCommand` as a simple wrapper of `spawn`

Reply:
Following on this. If optimisation fails should we clean up the state by removing the video so we don't accidentally end  up using the unoptimised video?